### DISCUSSION TITLE
Ports used

### DISCUSSION THREAD

**Discussion by loll45** [dbarzin/mercator#778] [Q&A] [answered]
Bonjour ,
Tout d'abord un grand merci à dbarzin pour ce travail et ce partage ! Je suis en première année de BTS sio en stage je m'occupe de faire la cartographie de l'entreprise et je rencontre un problème. J'ai supprimé un lien dans l'infrastructure physique suite à une erreur d'attribution et depuis je ne peux plus assigner ce port à un autre équipement j'ai ce message 'source port already used', il est pourtant bien libre je ne comprends pas. Quelqu'un a déjà rencontré ce problème ? Avec vous une idée pour le résoudre ? Merci d'avance et désolé si c'est une simple bêtise de ma part je débute dans le métier, soyez indulgent 😅

**Comment by dbarzin** [comment]
Cela doit être un bug dans le code. Pour résoudre le problème rapidement, il va falloir aller modifier la valeur dans la table physical_links.
Une correction sera disponible rapidement.

**Reply by loll45** [reply]
Merci pour votre réponse et tellement rapide, je vais regarder ça sûrement en ligne de commande dans mysql donc 🤔

**Reply by dbarzin** [reply, marked as answer]
Tu peux essayer ceci : 

    php artisan db

    delete from plysical_links where deleted_at is not null;

Cela va supprimer tous les liens physiques qui sont supprimés logiquement de la DB.

**Reply by loll45** [reply]
Effectivement l'erreur n'est plus là, Merci beaucoup ! 

PS: petite erreur de frappe c'est bien physical_links et non plysical_links comme quoi ça arrive même aux meilleurs 😁 merci encore Didier !
